articles/azure/azs-how-create-S2S-VPN-ps.md

```diff
@@ -0,0 +1,188 @@
+| \$AzureVNetRange        | Address space of virtual network to create in public Azure in CIDR notation                | <form oninput="result.value=AzureVNetRange.value" id="AzureVNetRange" style="display: inline;" ><input  type="text" id="AzureVNetRange" name="AzureVNetRange" style="display: inline;" placeholder="10.2.0.0/16"/></form> |
+| \$AzureSubnetRange      | Address space of virtual network subnet to create in public Azure in CIDR notation         | <form oninput="result.value=AzureSubnetRange.value" id="AzureSubnetRange" style="display: inline;" ><input  type="text" id="AzureSubnetRange" name="AzureSubnetRange" style="display: inline;" placeholder="10.2.0.0/24"/></form> |
+| \$AzureGWSubnetRange    | Address space of virtual network gateway subnet to create in public Azure in CIDR notation | <form oninput="result.value=AzureGWSubnetRange.value" id="AzureGWSubnetRange" style="display: inline;" ><input  type="text" id="AzureGWSubnetRange" name="AzureGWSubnetRange" style="display: inline;" placeholder="10.2.1.0/24"/></form> |
+| \$AzurePublicIPName     | Name of public IP to create in public Azure                                                | <form oninput="result.value=AzurePublicIPName.value" id="AzurePublicIPName" style="display: inline;" ><input  type="text" id="AzurePublicIPName" name="AzurePublicIPName" style="display: inline;" placeholder="S2S-IP"/></form> |
+| \$AzureVirtualGWName    | Name of virtual network gateway to create in public Azure                                  | <form oninput="result.value=AzureVirtualGWName.value" id="AzureVirtualGWName" style="display: inline;" ><input  type="text" id="AzureVirtualGWName" name="AzureVirtualGWName" style="display: inline;" placeholder="S2S-VNG"/></form> |
+| \$AzureLocalGWName      | Name of local network gateway to create in public Azure                                    | <form oninput="result.value=AzureLocalGWName.value" id="AzureLocalGWName" style="display: inline;" ><input  type="text" id="AzureLocalGWName" name="AzureLocalGWName" style="display: inline;" placeholder="S2S-LNG"/></form> |
+| \$AzureGWConnectionName | Name of virtual network gateway connection to create in public Azure                       | <form oninput="result.value=AzureGWConnectionName.value" id="AzureGWConnectionName" style="display: inline;" ><input  type="text" id="AzureGWConnectionName" name="AzureGWConnectionName" style="display: inline;" placeholder="S2S-Connection"/></form> |
+| \$SharedKey             | Encryption key to use for VPN connection                                                   | <form oninput="result.value=SharedKey.value" id="SharedKey" style="display: inline;" ><input  type="text" id="SharedKey" name="SharedKey" style="display: inline;" placeholder="Password123!"/></form> |
+
+## Create site-to-site VPN connection from Azure Stack to Azure
```

Comment:
Can we make this a bit clearer? Is it from UKCloud for Microsoft Azure to Azure public?

Reply:
Have changed "Azure" to "public Azure" which I think clears things up?